bin/travis-ci/conf.malta64_qemu

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016 Konsulko Group. All rights reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+console_impl=qemu
+qemu_machine="malta"
+qemu_binary="qemu-system-mips64"
+qemu_extra_args="-nographic -cpu MIPS64R2-generic -m 256"
+qemu_kernel_args="-drive if=pflash,file=${U_BOOT_BUILD_DIR}/flash,format=raw -netdev user,id=net0,tftp=${UBOOT_TRAVIS_BUILD_DIR} -device pcnet,netdev=net0"
```

Comment:
Existing MIPS qemu don't have most of these cmdline arguments. Is adding them deliberate?
Can we wrap the text so it's a bit narrower; github diff doesn't show the whole text without scrolling:-(

Reply:
Yes, them are all needed. The memory has a fixed size of 256MiB, which is not the Qemu default. Also I had problems with the default Qemu CPUs for the Malta machine so I needed to overwrite them. The network stuff is needed to be able to run the network test cases.

I'll push a fixup for the line wrapping.

Reply:
No, the qemu_mips boards didn't have networking support.